hicommonwealth/commonwealth
# Remove unused chat-related code.

## Link to Issue
Closes: #4079

## Description of Changes
- Removes routes, database models, and websocket logic from the codebase for deprecated chat feature.
- Migration to update DB schemas (drop associated tables).

## Test Plan
- Verify migration functions as expected.
- Ensure site loads properly -- should have no impact.
